azure-pipelines.yml

```diff
@@ -215,6 +215,16 @@ jobs:
       TargetFolder: $(buildOutDirAnyCpu)\PowerShell\Microsoft.WinGet.Configuration\SharedDependencies\$(BuildPlatform)
       flattenFolders: true
 
+  - task: PowerShell@2

```

Comment:
This seems better to be in the PowerShell job, which only runs once (not per-architecture) and because it is explicitly PowerShell related.  It could also target the artifacts directory for its output so that one could download the artifact and test with the help included (at least, I would hope it could work that way).